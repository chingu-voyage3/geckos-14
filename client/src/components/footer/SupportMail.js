import React from 'react';

const SupportMail = () => {
   return (
       <div className="footer-supportMail">
        <i className="fa fa-envelope fa-x2"></i><link to="/contact">support@dashthings.io</link>

        </div>
    );
};

  export default SupportMail;
