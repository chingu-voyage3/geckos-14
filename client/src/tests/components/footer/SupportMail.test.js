import React from 'react';
import ReactDOM from 'react-dom';
import SupportMail from '../../../components/footer/SupportMail';
import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<SupportMail />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
