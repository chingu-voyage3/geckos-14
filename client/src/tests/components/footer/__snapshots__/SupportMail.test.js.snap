// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="footer-supportmail"
>
  <Link
    replace={false}
    to="/contact"
  >
    <i
      className="fa fa-envelope fa-x2"
    />
     support@dashthings.io
  </Link>
</div>
`;
